best-practices/setting_up_rails_projects.md

```diff
@@ -14,9 +14,8 @@ Use the [config](https://github.com/railsconfig/config) gem for configuration ma
 ## Continuous Integration
 Projects should have implemented continuous integration. The following services are used often:
 
- - [travis-ci](https://travis-ci.org/) for running test suites
- - [hound-ci](https://houndci.com/repos) for style checking
```

Comment:
@sul-dlss/access-team do you still prefer hound?

Reply:
Nope.. we are looking to 🔥